src/useq/_plate.py

```diff
@@ -362,11 +385,11 @@ def plot(self) -> None:
 
         _, ax = plt.subplots()
 
+        # hide axes
+        ax.axis("off")
```

Comment:
i left axes in there since it is useful for verifying that the coordinates were actually applied.  otherwise, there's no sense for scale (in microns) and it's hard to be sure that the proper translation/scaling is applied right?

if you want to be able to use this plot elsewhere and don't want those numbers, maybe add a parameter? or  return the axis object so that you could modify it later?